apps/journeys-admin/src/components/Editor/Slider/Content/Canvas/QuickControls/DeleteBlock/DeleteBlock.tsx

```diff
@@ -46,8 +47,7 @@ export function DeleteBlock({
     setOpenDialog(false)
     closeMenu?.()
   }
-
-  const disableAction = currentBlock == null || disabled
+  const disableAction = currentBlock == null || disabled || result?.loading
```

Comment:
Is result nullable? I would be expecting this to throw a lint error if it was. Since its not, it looks like its not nullable.
```suggestion
  const disableAction = currentBlock == null || disabled || result.loading
```